src/declarations.d.ts

```diff
@@ -1 +1,8 @@
-declare module 'redis-info'
+declare module 'redis-info' {
+  export const parse: (
+    rawRedisInfo: string,
+  ) => {
+    // This is not technically true, as metrics can also come as objects
```

Comment:
it's something 🙆‍♂

Reply:
I left this as it is because that's how we treat the return for this method currently. My fear of adding this type as it is, is that people will believe in it and treat object types wrongly. 😕 
I didn't want to make major changes to the codebase in this PR, rather just make it compatible with the typings.
Maybe it's a work for a follow-up? :D~

Reply:
yes! agreed, can you create a separate issue please @Yurickh?

Reply:
Will create an issue once https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42186 is merged. If you could take an extra look there as well to guarantee I'm not messing it up, that would be extra good! 🙏